projects/module-1/which-beyonce-group.md

```diff
@@ -0,0 +1,220 @@
+
+* **4:** Additional elements that have been added **match** the visuals established in the comps. **Finished touches** such as `cursor: pointer` and additional visual cues on hover are thoughtfully implemented.
+* **3:** Application implements all major comp details accurately and correctly on desktop and large TV. **Additional elements added generally match** the visuals established in the comps, but may be slightly awkward. **Nothing about the UI is confusing** to a user who has never seen this application before.
+* **2:** Application implements all major comp details accurately and correctly on desktop. Some UI may be slightly confusing or unclear.
+* **1:** Application implements most major comp details accurately and correctly on desktop.
+
+## HTML
+
+* **4:** Application uses BEM, SMACCS or another set of **naming conventions** for classes; all developers can explain those naming conventions. The form uses **WAI-ARIA** and is fully accessible.
+* **3:** Application uses markup that is **easy to read** and follows across **naming conventions**.
+* **2:** Application uses markup that incorporates **semantic HTML** elements and has a simple, clean HTML structure.
+* **1:** Application uses markup that meets the criteria of the [Turing HTML Style Guide](https://github.com/turingschool-examples/html).
+
+## JavaScript
+
+* **4:** **Code is well refactored** and demonstrates developer empathy. No global variables are used aside from query selectors, start time, and instances of `Player` and `Deck`. If you feel you need more because you are building out additional functionality that requires a global variable, please check in with an instructor. All functions are **less than 10 lines long** or otherwise demonstrate SRP.
+* **3:** The application correctly **implements a data model** for the `Card` and `Deck` classes, including all required methods. Arguments and parameters are used to limit global variables. **Function declarations** are used over anonymous functions in event listeners. Developers can speak to the **role of each class**. The separation of data model logic and presentational logic is clear and can be explained by all developers in the team.
+* **2:** The **event object** is used correctly, and is not accepted as a parameter if it is not necessary. Developer can speak to **how the event object is utilized** for any given event handler. **Function and variable names** describe their role in the program. *Examples: The name of the data type should not ever be in a variable name (ex: "petArray"); the name itself should be clear enough to indicate the type of data it holds (ex: "allPets"). An event handler should not have "handler" (ex: "clickHandler"); the name should indicate the handler's purpose (ex: "addNewPet").
+* **1:** Style and syntax meets the criteria of the [Turing JS Style Guide](https://github.com/turingschool-examples/javascript/tree/master/es5).
```

Comment:
I almost think that "***Function declarations** are used over anonymous functions in event listeners." can be moved to the reqs for a 1? 

Also making the 3 look a little more approachable.